react/react2/week1/search-app/git-app/src/components/SearchUser.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import User from './User';
+
+function SearchUser({ username }) {
+  const [isLoading, setIsloading] = useState(true);
+  const [user, setuser] = useState([]);
+
```

Comment:
you are missing state for the error. It was required (https://github.com/HackYourFuture-CPH/React/blob/main/react2/week1/homework.md)